Conformance language updates and refactoring (#29)
AMWA-TV/ms-05-01: docs/Definitions.md

**Control protocol** Definitions of message formats, data flows, data serializations, and exchange rules for accessing control APIs.

**Framework mechanism** Architectural mechanism essential to NCA operation in any type of device. Conformant Devices implement all of the Framework Mechanisms applicable.

> **Feature-set mechanism** opt-in mechanisms which devices implement if they implement the specific feature set.

Reviewer: Difficult one to word without using normative language...
```suggestion
**Feature-set mechanism** optional mechanism implemented by devices that implement a specific feature set.
```

Author: I was trying to avoid the word optional because feature sets should be seen as a requirement if devices implement that particular feature set. For example a device wishing to add receiver monitoring as a capability MUST either use the NcReceiverMonitor control class directly or derive it. They should not create their own completely separate model.

Reviewer: Understood. Difficult one as opt-in is synonymous with optional.  Would "additional" work better?

Author: Recommendation to change wording to:
**Feature-set mechanism** conditional mechanism required when a device implements a specific feature set.
